src/lib/flatten.js

```diff
@@ -0,0 +1,21 @@
+const flatten = (startingArray) => {
```

Comment:
As a matter of style, you may omit parentheses from single parameter functions in ES6:
```js
const flatten = startingArray => {
  /* body */
}
```